app/serializers/chapter_dates_serializer.rb

```diff
@@ -0,0 +1,22 @@
+class ChapterDatesSerializer
+
+  def initialize(dates)
+    @dates = dates
+  end
+
+  def serialize
+    @dates.keys.each do |chapter|
+      ['last_updated', 'next_updated'].each do |date_type|
+        chapter_dates = @dates[chapter]
+        date = Date.strptime(chapter_dates[date_type], '%b-%y')
```

Comment:
Here you can use the `to_date` method on the string rather than calling the `Date` module, so something like
`date = chapter_dates[date_type].to_date`